src/main/java/software/amazon/awssdk/crt/CrtResource.java

```diff
@@ -14,16 +14,20 @@
  */
 package software.amazon.awssdk.crt;
 
+import java.io.Closeable;
 import java.util.Collection;
 import java.util.Collections;
+import java.util.LinkedList;
 import java.util.concurrent.ConcurrentHashMap;
 
 /**
  * This wraps a native pointer to an AWS Common Runtime resource. It also ensures
  * that the first time a resource is referenced, the CRT will be loaded and bound.
  */
-public class CrtResource {
+public class CrtResource implements Closeable, AutoCloseable {
```

Comment:
You only need AutoCloseable. it is undesirable to use both, and AutoCloseable supercedes Closeable.

Reply:
Fixed